tests/test_complete_run.cfg

```diff
@@ -0,0 +1,98 @@
+[default]
+input = /lcrc/group/e3sm/ac.forsyth2/E3SM_simulations/20210122.v2_test01.piControl.ne30pg2_EC30to60E2r2-1900_ICG.chrysalis
```

Comment:
I was using my own Chrysalis run as input. Is there another input that would be a better test case?

Reply:
This is fine for now. We could update later when we have official v2 simulations.